CachingDirectivesExamplesSpec.createCache

```
org.scalatest.exceptions.TestFailedException: "...mple.com/1 @ count 1[]" did not equal "...mple.com/1 @ count 1[00]"
	at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:340)
	at org.scalatest.Matchers$AnyShouldWrapper.shouldEqual(Matchers.scala:6742)
	at docs.http.scaladsl.server.directives.CachingDirectivesExamplesSpec$$anonfun$14$$anonfun$apply$18.apply(CachingDirectivesExamplesSpec.scala:141)
	at docs.http.scaladsl.server.directives.CachingDirectivesExamplesSpec$$anonfun$14$$anonfun$apply$18.apply(CachingDirectivesExamplesSpec.scala:141)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
	at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
	at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.$tilde$greater(RouteTestResultComponent.scala:50)
	at docs.http.scaladsl.server.directives.CachingDirectivesExamplesSpec$$anonfun$14.apply(CachingDirectivesExamplesSpec.scala:140)
	at docs.http.scaladsl.server.directives.CachingDirectivesExamplesSpec$$anonfun$14.apply(CachingDirectivesExamplesSpec.scala:100)
```
https://jenkins.akka.io:8498/job/pr-validator-akka-http/1729/testReport/junit/docs.http.scaladsl.server.directives/CachingDirectivesExamplesSpec/createCache/

Is this the same problem except on Java?
```
java.lang.AssertionError: Unexpected entity! expected:<...mple.com/1 @ count 1[00]> but was:<...mple.com/1 @ count 1[]>
  Request was:      HttpRequest(HttpMethod(GET),/1,List(),HttpEntity.Strict(none/none,ByteString()),HttpProtocol(HTTP/1.1))
  Route result was: FulfilledFuture(Complete(HttpResponse(200 OK,List(),HttpEntity.Strict(text/plain; charset=UTF-8,Request for http://example.com/1 @ count 1),HttpProtocol(HTTP/1.1))))

	at akka.http.javadsl.testkit.JUnitRouteTestBase$$anon$1.reportDetails(JUnitRouteTest.scala:45)
	at akka.http.javadsl.testkit.JUnitRouteTestBase$$anon$1.assertEquals(JUnitRouteTest.scala:30)
	at akka.http.javadsl.testkit.TestRouteResult.assertEqualsKind(TestRouteResult.scala:227)
	at akka.http.javadsl.testkit.TestRouteResult.assertEntity(TestRouteResult.scala:166)
	at docs.http.javadsl.server.directives.CachingDirectivesExamplesTest.testCreateCache(CachingDirectivesExamplesTest.java:176)
```
https://jenkins.akka.io:8498/job/pr-validator-akka-http/1730/testReport/junit/docs.http.javadsl.server.directives/CachingDirectivesExamplesTest/testCreateCache/

--- a/docs/src/test/java/docs/http/javadsl/server/directives/CachingDirectivesExamplesTest.java
+++ b/docs/src/test/java/docs/http/javadsl/server/directives/CachingDirectivesExamplesTest.java
@@ -165,7 +165,7 @@ public class CachingDirectivesExamplesTest extends JUnitRouteTest {
       .run(HttpRequest.GET("/1"))
       .assertEntity("Request for http://example.com/1 @ count 1");
 
-    for (int i = 1; i < 100; i++) {
+    for (int i = 1; i < 500; i++) {
       testRoute(route)
         .run(HttpRequest.GET("/" + i))
         .assertEntity("Request for http://example.com/" + i + " @ count " + i);
@@ -173,6 +173,6 @@ public class CachingDirectivesExamplesTest extends JUnitRouteTest {
 
     testRoute(route)
       .run(HttpRequest.GET("/1"))
-      .assertEntity("Request for http://example.com/1 @ count 100");
+      .assertEntity("Request for http://example.com/1 @ count 500");
   }
 }

--- a/docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala
+++ b/docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala
@@ -131,14 +131,14 @@ class CachingDirectivesExamplesSpec extends RoutingSpec with CachingDirectives {
       responseAs[String] shouldEqual "Request for http://example.com/1 @ count 1"
     }
 
-    for (i <- 1 until 100) {
+    for (i <- 1 until 500) {
       Get(s"/$i") ~> route ~> check {
         responseAs[String] shouldEqual s"Request for http://example.com/$i @ count $i"
       }
     }
 
     Get("/1") ~> route ~> check {
-      responseAs[String] shouldEqual "Request for http://example.com/1 @ count 100"
+      responseAs[String] shouldEqual "Request for http://example.com/1 @ count 500"
     }
   }
 }

